.github/workflows/continuous.yml

```diff
@@ -38,10 +38,12 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
       - name: Build with Gradle
-        run: ./gradlew --refresh-dependencies build :jacoco:testCodeCoverageReport
```

Comment:
Why only build for python engine?
This workflow is used to verify the whole repo

Reply:
I was debugging the jacoco report error. It's not going to be commited.